### ISSUE TITLE
Raw DSL element needed

### ISSUE THREAD

**Issue by tautrimas** [ongr-io/ElasticsearchBundle#322]
When building a Query, sometimes we need a feature that is not supported by ESB yet. Some `RawXXX` is needed of form:

```php
$search->addAggregation(new RawAggregation([
    'name' => 'foo',
    'field' => 'field,
    'aggregations' => [/*...*/],
]));
```

Which would inject custom aggregation introduced to Elastisearch, for example, 2 days ago.

Other types of elements might be added too, like `RawFilter`, etc.

**Comment by saimaz** [comment]
Thanks for the idea, it will be implemented in v1.0.0

**Comment by saimaz** [comment]
Elasticsearch filters and queries are in DSL library, created issue there and referenced.
